scripts/build-glean-metadata.py

```diff
@@ -135,17 +135,18 @@ def etl_snake_case(line: str) -> str:
     return "_".join([w.lower() for w in words if w.strip()])[::-1]
 
 
-def get_looker_ping_explore_url(looker_namespaces, app_name, ping_name, table_name):
+def get_looker_ping_explore_url(
+    looker_namespaces, app_name, ping_name, table_name, app_channel=None
```

Comment:
Let's make this non-optional for now.

```suggestion
    looker_namespaces, app_name, ping_name, table_name, app_channel
```